hardware/raspberrypi/boot_diagnostics.md

```diff
@@ -0,0 +1,26 @@
+## Boot Diagnostics on the Raspberry Pi 4
+
+The bootloader on the Raspberry Pi 4 is stored in EEPROM, and as well as providing boot services, is also capable of providing diagnostic information on an attached HDMI display. To display this information, remove the SD card from the Raspberry Pi 4, and reboot. A diagnostic display similar to below should appear on the attached display.
+
+This diagnostics page will also appear if the bootloader is unable to boot from an inserted SD card, or is unable to network boot; for example, if there is no bootable image on the card, or it is defective, or the network boot parameters are incorrect.
+
+
+
+
+The top line describes the model of Pi and its memory capacity. The QR code is a link to the [Downloads Page](https://raspberrypi.org/downloads).
+
+The diagnostic information is as follows:
+
+| Line: | Information |
+| ---- | ----------- |
+| bootloader | Bootloader version - build date |
+| board      | Board revision - Serial Number - Ethernet MAC address | 
+| boot       | |
+| SD CID	   | |
+| part	     | |
+| fw	       | |
```

Comment:
Is it worth adding a brief description of what theses other lines mean too? Or should they be treated as "technical internal details" which are only of interest to Raspberry Pi Engineers when trying to diagnose problems?

Reply:
Sorry, just noticed that this PR is still marked as WIP!

Reply:
Should be there now.